test/slurm/job_conf.xml

```diff
@@ -26,7 +27,16 @@
           -->
           <param id="docker_volumes" from_environ="GALAXY_DOCKER_VOLUMES">$defaults</param>
         </destination>
+        <destination id="local" runner="local"/>
     </destinations>
+    <tools>
+        <!-- Tools can be configured to use specific destinations or handlers,
+             identified by either the "id" or "tags" attribute.  If assigned to
+             a tag, a handler or destination that matches that tag will be
+             chosen at random.
+         -->
+        <tool id="upload1" destination="local"/>
```

Comment:
The meme stuff can be ignored, I should have deleted those entries (the package in the toolshed works fine outside the container).